ansible_collections/arista/avd/roles/eos_designs/python_modules/connected_endpoints/utils.py

```diff
@@ -92,9 +92,10 @@ def _get_short_esi(self, adapter: dict, channel_group_id: int, short_esi: str =
             # Only configure ESI for multi-homing.
             return None
 
-        # short_esi is only set when called from sub-interface port-channels.
         if short_esi is None:
-            if (short_esi := get(adapter, "ethernet_segment.short_esi")) is None:
+            # Setting short_esi under port_channel will be deprecated in AVD5.0
```

Comment:
```suggestion
            # Setting short_esi under port_channel will be removed in AVD 5.0
```

Reply:
I am not sure why we need code changes here. We should only be deprecating.

Reply:
Fixed